CMakeLists.txt

```diff
@@ -114,16 +119,18 @@ if( DETRAY_SETUP_ACTSVG )
 endif()
 
 # Set up JSON for I/O
-option( DETRAY_SETUP_NLOHMANN
-   "Set up the nlohmann::json target(s) explicitly" TRUE )
-option( DETRAY_USE_SYSTEM_NLOHMANN
-   "Pick up an existing installation of nlohman::json from the build environment"
-   ${DETRAY_USE_SYSTEM_LIBS} )
-if( DETRAY_SETUP_NLOHMANN )
-   if( DETRAY_USE_SYSTEM_NLOHMANN )
-      find_package( nlohmann_json REQUIRED )
-   else()
-      add_subdirectory( extern/nlohmann_json )
+if( DETRAY_IO_JSON )
```

Comment:
This seems unnecessary. :thinking: You could tie the default value of `DETRAY_SETUP_NHLOHMANN` to the value of `DETRAY_IO_JSON`. I.e.

```cmake
option( DETRAY_SETUP_NLOHMANN
   "Set up the nlohmann::json target(s) explicitly" ${DETRAY_IO_JSON} )
```

But the `DETRAY_SETUP_NHLOHMANN` variable should still be used to decide whether anything should be done here...

Reply:
OK